<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>451</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>203</x>
       <y>0</y>
       <width>20</width>
       <height>371</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="browse_button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>0</y>
       <width>241</width>
       <height>381</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Explanation</string>
      </attribute>
      <widget class="QPlainTextEdit" name="explanation_textEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>241</width>
         <height>351</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Graph</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QTextEdit" name="datasetpath_textEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="dataset_path_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>             Dataset path input</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="results_label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Results</string>
     </property>
    </widget>
    <widget class="QPushButton" name="analyze_button">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Analyze</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>-7</x>
       <y>253</y>
       <width>221</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>210</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="dataset_path_label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Target Column</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="targetColumnLineEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Enter the name of the column from your dataset you want to analyze</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sensitiveAttributesLineEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Enter attributes that might lead to biased decisions or outcomes</string>
     </property>
    </widget>
    <widget class="QLabel" name="dataset_path_label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Sensitive Attributes</string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <zorder>tabWidget</zorder>
    <zorder>line</zorder>
    <zorder>browse_button</zorder>
    <zorder>datasetpath_textEdit</zorder>
    <zorder>dataset_path_label</zorder>
    <zorder>line_2</zorder>
    <zorder>results_label</zorder>
    <zorder>analyze_button</zorder>
    <zorder>plainTextEdit</zorder>
    <zorder>line_3</zorder>
    <zorder>dataset_path_label_2</zorder>
    <zorder>targetColumnLineEdit</zorder>
    <zorder>sensitiveAttributesLineEdit</zorder>
    <zorder>dataset_path_label_3</zorder>
    <zorder>line_4</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
